assets/js/search.js

```diff
@@ -37,7 +37,7 @@
         $('#search-results-count').text("0 repos");
       }
     } else {
-      if (totalCount > 0 && numSubmodules === 0) {
+      if (totalCount > 0 ) {
```

Comment:
Does this still work properly on IaC Lib page?

Reply:
this doesn't affect the iac lib page (if you expand the code above, you'll see this is in the `else` block, so numsubmodules is not relevant anymore, which is why I removed it